RELEASE-NOTES.md

```diff
@@ -1,5 +1,10 @@
 # Wikibase CodeSniffer standards changelog
 
+## 0.4.0 (dev)
+
+* Updated the base MediaWiki rule set from 15 to 16.0.0. This adds the following sniffs:
```

Comment:
15 → 15.0.0

Reply:
This is intentional. Where we are coming from is not that important any more at this point. It can be seen just a few lines below.

Reply:
I don’t know… since the composer.json required exactly `15.0.0` (not something like `^15.0.0` or `~15.0.0`), saying ”15.x” feels almost misleading to me…